src/data/pages/interpreting-hubble-plot-1.json

```diff
@@ -19,6 +19,7 @@
     "sources": [
       {
         "name": "ZTF19abqmpsr",
+        "color": "#1ab579",
```

Comment:
Also I think these colors should probably be applied to galaxies within the app and not within the data.  Probably in the container for all of our bits and bobs we can just add this field to the galaxies we're showing. and then pass that updated dataset down to any component that needs it.

Reply:
Wouldn't this restrict the CMS user to not be able to have the flexibility of changing the color representing the galaxies? I added the color prop to the json structure to be able to have it seemlessly carry over into the plotter since the plotted galaxy points will match the color in the sidebar and selection circles. If you truly feel that this functionality is too verbose then I'll be happy to add these colors as static colors into the containing component as suggested.